src/pages/iou/IOUModal.js

```diff
@@ -41,35 +55,52 @@ class IOUModal extends Component {
         this.navigateToNextStep = this.navigateToNextStep.bind(this);
         this.updateAmount = this.updateAmount.bind(this);
         this.currencySelected = this.currencySelected.bind(this);
-
+        this.createTransaction = this.createTransaction.bind(this);
+        this.updateComment = this.updateComment.bind(this);
         this.addParticipants = this.addParticipants.bind(this);
+
         this.state = {
             currentStepIndex: 0,
             participants: [],
             amount: '',
```

Comment:
We should comment amount, to let people know it is the currency in decimal format